features/step_definitions/sections_steps.rb

```diff
@@ -1,21 +1,27 @@
 # frozen_string_literal: true
 
 Then('I can see a collection of sections') do
-  @test_site.section_experiments.search_results.each_with_index do |search_result, i|
-    expect(search_result.title.text).to eq("title #{i}")
+  @test_site
```

Comment:
For now can we not do this, I think it looks messy. Is there something else we could do? Maybe assigning out the section beforehand to a local variable?